syntax = "proto3";

package fixbug;

option cc_generic_services = true;

message ResultCode{
    int32 errcode = 1;
    bytes errmsg = 2;
}


message GetFriendsListRequest{
    int32 uid = 1;
};

message GetFriendsListResponse{
    ResultCode result = 1;
    bool sucess = 2;
    repeated bytes friends = 3;
}

service FriendServiceRpc{
    rpc GetFriendsList(GetFriendsListRequest) returns(GetFriendsListResponse);
}